android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java

```diff
@@ -218,23 +218,21 @@ protected void doInBackgroundGuarded(Void... params) {
           for (int idx=0; idx < keyValueArray.size(); idx++) {
             if (keyValueArray.getArray(idx).size() != 2) {
               error = AsyncStorageErrorUtil.getInvalidValueError(null);
-              return;
```

Comment:
Won't we risk overwriting errors now? Should they be concatenated? Should we throw instead?